### PR TITLE
Reformatting GIFTs app

### PR DESCRIPTION
**Pull request by james-monkeyshines**
The GIFTs service now has the same formatting as other Production services, with a navbar along the top, and defaulting to the submission page. The jobs table needs much less complexity than other services, so it has a somewhat stripped-down feel.

Also changed the colour scheme, green was (appropriately) bagged by the Plant services, but nothing is using orange...

### CODE REVIEW

**Comment by james-monkeyshines** [comment]
Screenshot from 2021-08-19 17-23-17
Screenshot from 2021-08-19 17-23-51

File: src/ensembl/production/gifts/templates/submit.html [hunk 1]
```diff
@@ -1,17 +1,25 @@
 
 {% extends 'base.html' %}
 
-{% set title = 'GIFTs: Submit Job' %}
 {% set icon_image = 'img/gifts.png' %}
+{% set title = 'GIFTs Service' %}
```
Comment (marcoooo) [inline src/ensembl/production/gifts/templates/submit.html:5]: For the icon image, I think it's best to use absolute path.

**Comment by marcoooo** [comment]
> Screenshot from 2021-08-19 17-23-17
> Screenshot from 2021-08-19 17-23-51

Awesome!

File: src/ensembl/production/gifts/templates/submit.html (hunk 1 above)
Comment (james-monkeyshines) [reply src/ensembl/production/gifts/templates/submit.html:5]: An absolute path is created when this variable is passed to the `url_for` method, it ends up being `/static/img/gifts.png`

Comment (marcoooo) [reply src/ensembl/production/gifts/templates/submit.html:5]: sounds about right. btw, regarding upcoming ENSPROD-7097 doing, If you have any JS associated somewhere which rely on absolute paths, please tell me, we'll discuss how to get rid of it!

**Pull request merged**